zookeeper-client/zookeeper-client-c/tests/TestMulti.cc

```diff
@@ -656,6 +667,63 @@ class Zookeeper_multi : public CPPUNIT_NS::TestFixture
         waitForMultiCompletion(5);
      }
 
+     /**
+     * ZOOKEEPER-1636: If request is too large, the server will cut the
+     * connection without sending response packet. The client will try to
+     * process completion on multi request and eventually cause SIGSEGV
+     */
+    void testBigAsyncMulti() {
+        int rc;
+        int callback_rc = (int) ZOK;
+        watchctx_t ctx;
+        zhandle_t *zk = createClient(&ctx);
+
+         // The request should be more than 1MB which exceeds the default
+        // jute.maxbuffer and causes the server to drop client connection
+        const int iteration = 500;
+        const int type_count = 3;
+        const int nops = iteration * type_count;
+        char buff[1024];
+
+         zoo_op_result_t results[nops];
```

Comment:
There're some indentation mistakes in this test. Please fix them.

Reply:
done!

// thank you for this catch!